sink/datadog.go

```diff
@@ -0,0 +1,251 @@
+package sink
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/DataDog/datadog-api-client-go/v2/api/datadog"
+	"github.com/DataDog/datadog-api-client-go/v2/api/datadogV2"
+
+	"github.com/cashapp/blip"
+	"github.com/cashapp/blip/sink/tr"
+	"github.com/cashapp/blip/status"
+)
+
+// SignalFx sends metrics to SignalFx.
+type DataDog struct {
+	monitorId string
+	dim       []string            // monitor.tags (dimensions)
```

Comment:
If Datadog calls these tags, let's rename the var because "dim" is short for "dimension" which is what SFX calls them.